socialhome/users/serializers.py

```diff
@@ -51,10 +60,14 @@ class Meta:
             "location",
             "name",
             "nsfw",
+            "pinned_content_exists",
+            "safer_image_url_large",
```

Comment:
No need for `safer_*` image url method usage any more - those will soon be removed, since #348. And the API already has `image_url_large`.